### PR TITLE
Convert html to markdown. Remove obsolete stuff

### PR DESCRIPTION
**Pull request by manolo**
[Review on Reviewable](https://reviewable.io/reviews/gwtproject/gwt-site/104)

### CODE REVIEW

File: src/main/markdown/doc/latest/DevGuideI18n.md [hunk 1]
```diff
@@ -321,24 +297,18 @@ different keys, as they may be translated differently.</li>
...
-solving the problem of reading UTF-8 properties files in Tapestry's `LocalizedProperties` class.
+Both [Constants](/javadoc/latest/com/google/gwt/i18n/client/Constants.html) and [Messages](/javadoc/latest/com/google/gwt/i18n/client/Messages.html) use traditional Java properties files, with one notable difference: properties files used with GWT should be encoded as UTF-8 and may contain Unicode characters directly, avoiding the need for `native2ascii`. See the
+API documentation for the above interfaces for examples and formatting details.
 
 In order to use internationalized characters, make sure that your host HTML file contains the `charset=utf8` content type in the meta tag in the header:
 
-```
-<meta http-equiv="content-type" content="text/html;charset=utf-8" />
-```
+    <meta http-equiv="content-type" content="text/html;charset=utf-8" />
```
Comment (tbroyer) [inline src/main/markdown/doc/latest/DevGuideI18n.md]: Should we change this to `<meta charset=utf-8>`?

File: src/main/markdown/doc/latest/DevGuideI18n.md [hunk 2]
```diff
@@ -321,24 +297,18 @@ different keys, as they may be translated differently.</li>
...
-one notable difference: properties files used with GWT should be encoded as UTF-8 and may contain Unicode characters directly, avoiding the need for `native2ascii`. See the
-API documentation for the above interfaces for examples and formatting details. Many thanks to the [Tapestry](http://tapestry.apache.org/) project for
-solving the problem of reading UTF-8 properties files in Tapestry's `LocalizedProperties` class.
+Both [Constants](/javadoc/latest/com/google/gwt/i18n/client/Constants.html) and [Messages](/javadoc/latest/com/google/gwt/i18n/client/Messages.html) use traditional Java properties files, with one notable difference: properties files used with GWT should be encoded as UTF-8 and may contain Unicode characters directly, avoiding the need for `native2ascii`. See the
+API documentation for the above interfaces for examples and formatting details.
 
 In order to use internationalized characters, make sure that your host HTML file contains the `charset=utf8` content type in the meta tag in the header:
```
Comment (tbroyer) [inline src/main/markdown/doc/latest/DevGuideI18n.md]: utf8 → utf-8 as we're at it.

**Comment by tbroyer** [comment]
LGTM with nits.

File: src/main/markdown/doc/latest/DevGuideI18n.md (hunk 1 above)
Comment (manolo) [inline src/main/markdown/doc/latest/DevGuideI18n.md]: Very right : https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Obsolete_things_to_avoid

File: src/main/markdown/doc/latest/DevGuideI18n.md (hunk 2 above)
Comment (manolo) [inline src/main/markdown/doc/latest/DevGuideI18n.md]: Yes, sounds better

**Comment by tbroyer** [comment]
LGTM

(merge, or fast-forward, and/or squash at your pleasure)

**Pull request merged**